packages/vite-plugin-lib/package.json

```diff
@@ -18,20 +18,18 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
       "require": "./dist/index.cjs",
       "import": "./dist/index.mjs"
     }
   },
   "main": "dist/index.cjs",
   "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
   "files": [
     "dist",
     "LICENSE"
   ],
   "scripts": {
-    "build": "unbuild",
+    "build": "unbuild && cp dist/index.d.ts dist/index.d.mts && mv dist/index.d.ts dist/index.d.cts",
```

Comment:
couldnt figure out how in unbuild to rename the dts files so i just did it like this. i moved this copied files into my repo and it picked up on them appropriately.

I believe this is necessary because esm and cjs can't reference the same d.ts files. if you have them with the proper extension its automatically picked up though.

Reply:
Theres actually an issue with this in dts plugin because of api extractor.

https://github.com/microsoft/rushstack/issues/4214

https://github.com/microsoft/rushstack/issues/3116

Reply:
See above comment, this should be reverted, as everything works.